data/onPostBuild/transpileMdxToMarkdown.test.ts

```
@@ -442,9 +503,12 @@ import Baz from 'qux';
     });
 
     it('should not add .md to URLs that already have a file extension (.png)', () => {
-      const input = '[Image](https://raw.githubusercontent.com/ably/docs/main/src/images/content/diagrams/test.png)';
+      const input =
+        '[Image](https://raw.githubusercontent.com/ably/docs/main/src/images/content/diagrams/test.png)';
```

ESLint

[error] Delete `⏎·······`

(prettier/prettier)